docs/docs-beta/docs/guides/external-systems/apis.md

```diff
@@ -1,4 +1,53 @@
 ---
 title: Connecting to APIs
 sidebar_position: 20
----
\ No newline at end of file
+---
+
+This guide describes how to connect to and interact with APIs in dagster.
+
+
+## What you'll learn
+
+- How to write a dagster Resource to connect to an API
+- How to use that Resource in an asset
```

Comment:
```suggestion
- How to use the resource in an asset
```

Nit - `resource` doesn't need to be capitalized

Reply:
I see this occurs throughout the guide, so I won't pollute the review with a bunch of comments about it. Mind doing a find/replace to make it lowercase where appropriate?